hack/run-locally.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash 
+
+# run-locally.sh is some scaffolding around running a local instance of the
+# network operator with the installer.
+# See INSTALLER-HACKING.md
+
+set -o errexit
+set -o nounset
+
+# Install our overrides so the cluster doesn't run the network operator.
+function override() {
+    if [[ ! -e ${CLUSTER_DIR}/manifests/cvo-overrides.yaml ]]; then
+        echo "cannot find cvo-overrides.yaml; please run"
+        echo "openshift-install --dir=${CLUSTER_DIR} create manifests"
+        exit 1
+    fi
+
+    # Patch the CVO to not create the network operator
+    echo "Applying overrides to ${CLUSTER_DIR}/manifests/cvo-overrides.yaml"
+
+    kubectl --kubeconfig=hack/null-kubeconfig patch --type=json --local=true -f=${CLUSTER_DIR}/manifests/cvo-overrides.yaml -p "$(cat hack/overrides-patch.yaml)" -o yaml > ${CLUSTER_DIR}/manifests/.cvo-overrides.yaml.new
+
+    mv ${CLUSTER_DIR}/manifests/.cvo-overrides.yaml.new ${CLUSTER_DIR}/manifests/cvo-overrides.yaml
+}
+
+# Extract the image references from the release image.
+function build_env() {
+    echo "Writing image references to ${CLUSTER_DIR}/env.sh"
+    oc --kubeconfig=hack/null-kubeconfig convert --local=true -f manifests/0000_07_cluster-network-operator_03_daemonset.yaml -ojsonpath='{range .spec.template.spec.containers[0].env[?(@.value)]}{.name}{"="}{.value}{"\n"}' > ${CLUSTER_DIR}/env.sh
+}
+
+# start_operator waits for the cluster to come up, then launches
+# the operator locally.
+function start_operator() {
+    export KUBECONFIG=${CLUSTER_DIR}/auth/kubeconfig
+
+    echo "Waiting for $KUBECONFIG"
+    while true; do 
+        if [[ ! -e ${KUBECONFIG} ]]; then
+            sleep 10
+        else
+            break
+        fi
+    done
+
+    # A few environment variables we set in the Daemonset
+    export POD_NAME=LOCAL
+    export KUBERNETES_SERVICE_PORT=6443
+    export KUBERNETES_SERVICE_HOST=$(oc config view -o jsonpath='{.clusters[0].cluster.server}' | awk -F'[/:]' '{print $4}')
+
+    echo "Waiting for the apiserver to come up and for the network configuration to be rceated."
+    echo "You can ignore the error message 'error: the server doesn't have a resource type \"Network\"'"
```

Comment:
You can ignore the error message 'error: the server doesn't have a resource type "Network"'
    The connection to the server danwinship-api.devcluster.openshift.com:6443 was refused - did you specify the right host or port?

Anyway, we can make further tweaks to the instructions later